start.js

```diff
@@ -65,7 +65,7 @@
    * Keep track of processes, and clean up on SIGINT
    */
   function monitor () {
-    fs.writeFileSync(pidFile, process.pid, fileOptions)
+    fs.writeFileSync(pidFile, parseInt(process.pid).toString(), fileOptions)
```

Comment:
Very minor again but I'm not sure why this needs to be run through `parseInt`
```suggestion
    fs.writeFileSync(pidFile, process.pid.toString(), fileOptions)
```